server/src/main/java/org/elasticsearch/index/mapper/GeoPointFieldMapper.java

```diff
@@ -301,34 +301,25 @@ public void parse(ParseContext context) throws IOException {
                 XContentParser.Token token = context.parser().currentToken();
                 if (token == XContentParser.Token.START_ARRAY) {
                     token = context.parser().nextToken();
-                    if (token == XContentParser.Token.START_ARRAY) {
```

Comment:
When I said that we don't need to support MULTIPOINT at this time, I meant that we don't need to support new formats. But we cannot just remove formats that are already supported like this one.

Reply:
I didn't actually change any logic here. This code was redundant.
(Unless I missed something).

Reply:
What was removed will be done anyway. Take a look below, starting from line 324.